frontend/packages/dev-console/src/components/import/ImportStrategySection.tsx

```diff
@@ -59,6 +61,9 @@ const ImportStrategySection: React.FC<ImportStrategySectionProps> = ({ builderIm
         filePath: docker?.dockerfilePath,
       }),
       [ImportStrategy.S2I]: t('devconsole~A Builder Image is recommended.'),
+      [ImportStrategy.PAC]: t(
+        'devconsole~A Pipeline-as-Code file in .tekton directory is recommended.',
```

Comment:
Should we change it to `Pipelines-as-code` like it is mentioned in [ Pipelines as code docs](https://pipelinesascode.com/docs/install/github_apps/#create-a-pipelines-as-code-github-app) ?